### ISSUE TITLE
Signal Doesn't Start

### ISSUE THREAD

**Issue by maymage** [flathub/org.signal.Signal#287]
Hi, 

so the latest update to Signal Version 

```
flatpak info org.signal.Signal 

Signal Desktop - Private messenger

         Kennung: org.signal.Signal
             Ref: app/org.signal.Signal/x86_64/stable
     Architektur: x86_64
           Zweig: stable
         Version: 5.37.0
         License: GPL-3.0
        Ursprung: flathub
        Sammlung: org.flathub.Stable
    Installation: system
     Installiert: 372,4 MB
Laufzeitumgebung: org.freedesktop.Platform/x86_64/21.08
             Sdk: org.freedesktop.Sdk/x86_64/21.08

          Commit: d3eeb3ae608b7c69ec474b4b32bbfd925b1f5d91524f023…
          Parent: 767b8d7e311c8722344bfb91959f65c20737c3178a83bf7…
         Subject: Change socket to fallback-x11 (#285) (37a73516)
            Date: 2022-03-31 07:11:57 +0000
```

doesn't start anymore. Resetting flatpak permissions vial flatseal I get

```
flatpak run org.signal.Signal 
[2 preload-host-spawn-strategy] Running: /app/bin/zypak-helper child - /app/Signal/signal-desktop --type=zygote --enable-crashpad
Set Windows Application User Model ID (AUMID) { appUserModelId: 'org.whispersystems.signal-desktop' }
NODE_ENV production
NODE_CONFIG_DIR /app/Signal/resources/app.asar/config
NODE_CONFIG {}
ALLOW_CONFIG_MUTATIONS undefined
HOSTNAME undefined
NODE_APP_INSTANCE undefined
SUPPRESS_NO_CONFIG_WARNING undefined
SIGNAL_ENABLE_HTTP undefined
userData: /home/martin/.var/app/org.signal.Signal/config/Signal
config/get: Successfully read user config file
config/get: Successfully read ephemeral config file
LaunchProcess: failed to execvp:
xdg-settings
LaunchProcess: failed to execvp:
xdg-settings
[2:0331/103242.504047:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: Datei oder Verzeichnis nicht gefunden
[2:0331/103242.504217:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: Datei oder Verzeichnis nicht gefunden
[54:0331/103242.669897:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
[2:0331/103242.958787:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: Datei oder Verzeichnis nicht gefunden
[2:0331/103242.958925:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: Datei oder Verzeichnis nicht gefunden
[2:0331/103242.959033:ERROR:bus.cc(397)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: Datei oder Verzeichnis nicht gefunden
```

---

System info: 

* debian sid, current GNOME 42.0
* xdg-utils/unstable,now 1.1.3-4.1

**Comment by maymage** [comment]
Addendum: 

Granting `socket=system-bus`via flatseal leaves me with 

```
flatpak run org.signal.Signal 
[2 preload-host-spawn-strategy] Running: /app/bin/zypak-helper child - /app/Signal/signal-desktop --type=zygote --enable-crashpad
Set Windows Application User Model ID (AUMID) { appUserModelId: 'org.whispersystems.signal-desktop' }
NODE_ENV production
NODE_CONFIG_DIR /app/Signal/resources/app.asar/config
NODE_CONFIG {}
ALLOW_CONFIG_MUTATIONS undefined
HOSTNAME undefined
NODE_APP_INSTANCE undefined
SUPPRESS_NO_CONFIG_WARNING undefined
SIGNAL_ENABLE_HTTP undefined
userData: /home/martin/.var/app/org.signal.Signal/config/Signal
config/get: Successfully read user config file
config/get: Successfully read ephemeral config file
LaunchProcess: failed to execvp:
xdg-settings
LaunchProcess: failed to execvp:
xdg-settings
[63:0331/104430.060064:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
```

**Comment by maymage** [comment]
I confirm... removing the permission for `fallback-x11` makes Signal work again as expected.

**Comment by Vinnl** [comment]
I think this was fixed by #285.

**Comment by sebastian-de** [comment]
I assume `--socket=fallback-x11` should work as soon as v5.38.0 is released, since it contains an [update to electron 17.3.1](https://github.com/signalapp/Signal-Desktop/commit/a8962c7d20df8ff084749730b3114f6280053218), which should [fix several issues on Wayland](https://github.com/electron/electron/pull/33499).
